openwisp_monitoring/device/apps.py

```diff
@@ -166,12 +166,12 @@ def register_notification_types(self):
             },
         )
         register_notification_type(
-            'connection_is_not_working',
+            'connection_is_working',
             {
-                'verbose_name': 'Device  Alert (Connection not working)',
```

Comment:
@pandafy @nepython I switched the order of the two, put is not working first and working afterwards + changed the `verbose_name` to be consistent with the rest of the notification types.